### ISSUE TITLE
Clustering Documentation

### ISSUE THREAD

**Issue by sakshi012000** [Algo-Phantoms/Algo-ScriptML#121]
@ashwani-rathee I am a GSSoC participant. I want to make documentation in clustering in machine learning. Please assign me this.

**Comment by ashwani-rathee** [comment]
@geekquad Do we need this?

**Comment by geekquad** [comment]
Hey @sakshi012000, which clustering method are you willing to document?

**Comment by sakshi012000** [comment]
@geekquad  I will prefer partitioning methods. I can do any other also as per your preference.

**Comment by ashwani-rathee** [comment]
No progress on this

**Comment by sameeksharl** [comment]
heyy @geekquad @ashwani-rathee  i want to take up this issue could u pls assign it to me ?? I will be able to  finish the work by 2 -3 days??

**Comment by Abheeshth** [comment]
Hello, I am a GSSOC'21 participant Can I work on this issue??
@ashwani-rathee

**Comment by sameeksharl** [comment]
What kind of clustering can I work on ?? K means clustering??

**Comment by sameeksharl** [comment]
@geekquad  Could you please tell me what types of partitional clustering is needed to be part of documentation as there are many types ??
can i include commonly used types such as k means, k-medoids, CLARA

**Comment by Manasi2001** [comment]
Can I work on this @yukti845 @geekquad?

**Comment by sameeksharl** [comment]
No I am still working on it ,,will make pr by tomorrow  @geekquad

**Comment by Dishita2602** [comment]
I can work and submit all PR today itself

**Comment by Manasi2001** [comment]
> Can I work on this @yukti845 @geekquad?

@geekquad @ashwani-rathee please assign it to me so that I can start right away.

**Comment by geekquad** [comment]
No updates @sameeksharl.
Transferring this to @Dishita2602,

**Comment by Manasi2001** [comment]
> No updates @sameeksharl.
> Transferring this to @Dishita2602,

@geekquad why not assign it to me? I asked for it 3 days ago.

**Comment by Dishita2602** [comment]
> No updates @sameeksharl.
> Transferring this to @Dishita2602,

Okay I will make PR today itself

**Comment by geekquad** [comment]
Hey @Dishita2602, sorry but @Manasi2001 asked it before you and I missed it. 
Please look if you can contribute to something else.

**Comment by Dishita2602** [comment]
> Hey @Dishita2602, sorry but @Manasi2001 asked it before you and I missed it.
> Please look if you can contribute to something else.

Yeah I totally understand @Manasi2001 All the best ! Hope you make up to top 50

**Comment by Dishita2602** [comment]
@geekquad please assign me #201 I have made PR of codebase and tomorrow is last day of GSSoC'21 so asap
